lib/structures/Message.js

```diff
@@ -56,7 +56,7 @@ function parseMetadataModelSubmit(client, data) {
       baseData.triggeringInteractionMetadata = parseMetadataApplicationCommand(client, data.triggering_interaction_metadata);
       break;
     default: // type message component
-      baseData.triggeringInteractionMetadata = parseMetadataApplicationCommand(client, data.triggering_interaction_metadata);
+      baseData.triggeringInteractionMetadata = parseMetadataModelSubmit(client, data.triggering_interaction_metadata);
```

Comment:
I believe it must be `parseMetadataMessageComponent`, otherwise, `parseMetadataModelSubmit` will be recursive

Reply:
Right! Sorry, I've been rushing a lot 😓

Reply:
No problem. Take your time 👍